python/paddle/fluid/tests/book/high-level-api/recognize_digits/CMakeLists.txt

```diff
@@ -2,6 +2,16 @@ file(GLOB TEST_OPS RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}" "test_*.py")
 string(REPLACE ".py" "" TEST_OPS "${TEST_OPS}")
 
 # default test
-foreach(src ${TEST_OPS})
-    py_test(${src} SRCS ${src}.py)
-endforeach()
+if(NOT APPLE)
+    foreach(src ${TEST_OPS})
+        py_test(${src} SRCS ${src}.py)
+    endforeach()
+else()
+    foreach(src ${TEST_OPS})
+        if(${src} STREQUAL "test_recognize_digits_conv")
```

Comment:
are these all random fail test in Mac?

Reply:
only recognize_digits_conv, and I only disable this one in recognize_digits tests

Reply:
What's wrong with this unit-test. Could you paste some logs?

Reply:
test_recognize_digits_conv will randomly cause segment fault on Mac:

<img width="1270" alt="2018-09-28 5 13 51" src="https://user-images.githubusercontent.com/22361972/46199570-fb3a6980-c341-11e8-87a2-9b980df5e5fd.png">